android/app/src/main/java/com/rctwebrtcdemo/MainApplication.java

```diff
@@ -1,14 +1,12 @@
 package com.rctwebrtcdemo;
 
 import android.app.Application;
-import android.util.Log;
 
 import com.facebook.react.ReactApplication;
-import com.facebook.react.ReactInstanceManager;
 import com.facebook.react.ReactNativeHost;
 import com.facebook.react.ReactPackage;
 import com.facebook.react.shell.MainReactPackage;
-import com.oney.WebRTCModule.WebRTCModulePackage;
+import com.facebook.soloader.SoLoader;
```

Comment:
oops, seems you killed `com.oney.WebRTCModule.WebRTCModulePackage` therefore `cannot find symbol`